test/common/router/route_fuzz_test.cc

```diff
@@ -37,7 +51,68 @@ cleanRouteConfig(envoy::config::route::v3::RouteConfiguration route_config) {
   return clean_config;
 }
 
-// Check configuration for size and unimplemented options.
+bool validateMatcherConfig(const xds::type::matcher::v3::Matcher& matcher);
+
+bool validateOnMatchConfig(const xds::type::matcher::v3::Matcher::OnMatch& on_match) {
+  if (on_match.has_matcher() && !validateMatcherConfig(on_match.matcher())) {
+    return false;
+  }
+  if (on_match.action().typed_config().type_url().find("envoy.config.route.v3.Route") !=
+      std::string::npos) {
+    envoy::config::route::v3::Route on_match_route_action_config;
+    MessageUtil::unpackTo(on_match.action().typed_config(), on_match_route_action_config);
```

Comment:
This call may throw an EnvoyException. The call to `validateConfig(input)` in line 137 should move into the try-catch scope.

Reply:
done